<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>nbaArenas</class>
 <widget class="QWidget" name="nbaArenas">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTableView" name="arenaTV">
   <property name="geometry">
    <rect>
     <x>6</x>
     <y>94</y>
     <width>641</width>
     <height>351</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="arenaButton">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>50</y>
     <width>111</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>View all by arena</string>
   </property>
  </widget>
  <widget class="QPushButton" name="stadiumCapacityButton">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>50</y>
     <width>181</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>View all by stadium capacity</string>
   </property>
  </widget>
  <widget class="QLabel" name="totalLabel1">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>50</y>
     <width>97</width>
     <height>37</height>
    </rect>
   </property>
   <property name="text">
    <string>Grand Total Display</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>30</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Total Stadium Capacity</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QComboBox" name="arenaCB">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>171</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>6</x>
     <y>6</y>
     <width>481</width>
     <height>37</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 20pt &quot;MS Shell Dlg 2&quot;;
background : rgb(170, 170, 255)
</string>
   </property>
   <property name="text">
    <string>Arenas and Seating Capacity </string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
